modules/portmaster/src/i18n/helptexts.yaml

```diff
@@ -1,22 +1,23 @@
-############
-### Examples
-
-myKey:
-  title: Tipup Example
-  content: |
-    This is the Markdown formatted content.
-
-    This is a super cool, new feature that you will love!
-    It even supports markdown features like:
-    - order lists
-    - with multiple items
-
-    And :rocket: emojis
-
-    ### :tada: :facepalm:
-  url: https://docs.safing.io
-  urlText: Show me!
-  nextKey: navMonitor
+#############
+### Example
+#############
+# 
+# myKey:
+#   title: Tipup Example
+#   content: |
+#     This is the Markdown formatted content.
+# 
+#     This is a super cool, new feature that you will love!
+#     It even supports markdown features like:
+#     - order lists
+#     - with multiple items
+# 
+#     And :rocket: emojis
+# 
+#     ### :tada: :facepalm:
+#   url: https://docs.safing.io
+#   urlText: Show me!
+#   nextKey: navMonitor
```

Comment:
The example is only displayed in the developer mode (Due to being added to the add widget box) and is great for testing things. Would rather leave it in there.